app/models/diffusion_history_status.rb

```diff
@@ -1,5 +1,69 @@
 class DiffusionHistoryStatus < ApplicationRecord
   belongs_to :diffusion_history
 
+  before_validation :format_unsuccessful_reasons
+  validates :status, :diffusion_history_id, presence: :true
+  validate :is_valid_unsuccessful_reason
+  validate :has_unsuccessful_reasons_other
+  before_save :clear_unsuccessful_reasons
   # status strings are In Progress/Complete/Unsuccessful
+
+  ADOPTION_REASONS = {
+    0 => 'Lack of sufficient leadership/key stakeholder buy-in',
+    1 => 'Centralized decision-making that hinders ability to complete tasks (i.e. bureaucracy)',
+    2 => 'Unable to obtain necessary approvals',
+    3 => 'Unable to obtain necessary resources',
+    4 => 'Unable to attain necessary staff',
+    5 => 'Other'
+  }
+
+  ADOPTION_REASON_VALUES = ADOPTION_REASONS.keys
+
+  STATUSES = {
```

Comment:
Thoughts? We can use this constant instead of constantly renaming the status whenever we change our minds hahah 😅

Reply:
I like this idea. Who knows when we'll change the verbiage again haha